src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
Afaik legacy has its own legacy branch (and legacy-history branch too), while  "Socially Merge V2_0_0" cannot be retrieved from any branch anymore, but instead it has its own tag generated by Tortilla. There are also several older tags of "Socially merge" which are not present in the website. If you want to keep different versions of the same tutorial on the website you should add the abilitity to checkout tags, otherwise we should keep branching the tutorial each time is drastically changes (like what happened between legacy and "socially merge") while also avoiding to list different versions of the same tutorial.